site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -223,7 +229,18 @@ The number of rules available after that depends on how many rules are enabled b
 
 #### Dynamic and session rules {: #dynamic-session-rules }
```

Comment:
How about splitting this up into "Static rules", "Dynamic rules" and "Session rules". It's a bit unclear why static rules are standalone, but session and dynamic rule limits are together.

Reply:
How about describing the pre 121 behavior in a separate callout?

Reply:
I've moved things around a bit. Let me know what you think. Unfortunately a little tricky with changes in both Chrome 120 and Chrome 121 but tried to find a good compromise.

Reply:
Much better - thanks!